opent2t.base-schemas/opent2t.platform-schema.json

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
Fixed.

---
In reply to: [106228995](https://github.com/openT2T/translators/pull/197#discussion_r106228995) [](ancestors = 106228995)